### PR TITLE
Adding /bin to the PATH in ktest

### PR DESCRIPTION
**Pull request by radumereuta**
so test work even if the user didn't update his environment.

@dwightguth please review.

### CODE REVIEW

**Comment by andreistefanescu** [comment]
did you try a fresh installation on a machine without a changed `PATH`?

**Comment by radumereuta** [comment]
No.

**Comment by radumereuta** [comment]
Tested without the path, and it works as expected. Can we merge this?

**Comment by andreistefanescu** [comment]
Thanks! You can merge.

File: kernel/src/main/java/org/kframework/ktest/Proc.java [hunk 1]
```diff
@@ -141,9 +142,20 @@ public Proc(T obj, String[] args, File inputFile, String procInput,
...
         success = options.dry;
         this.warnings2errors = warnings2errors;
+        // add the /bin directory to the PATH if not there already
+        String binDir = ExecNames.getBinDirectory();
+        // windows is case insensitive w.r.t environment variables, so first find the exact match
+        String path = this.env.keySet().stream().filter(key -> key.toUpperCase().equals("PATH")).findFirst().orElseGet(() -> "PATH");
+        if (this.env.containsKey(path)) {
+            String allPaths = this.env.get(path);
+            if (!allPaths.contains(binDir))
+                this.env.put(path, allPaths + ";" + binDir);
```
Comment (dwightguth) [inline kernel/src/main/java/org/kframework/ktest/Proc.java:155]: you need to use File.pathSeparator instead of ";" since it is separated by ":" on linux.

**Comment by radumereuta** [comment]
Ah, I will create a new pull request in just a sec.

**Pull request merged**